arduino-ide-extension/package.json

```
@@ -163,11 +163,7 @@
   ],
   "arduino": {
     "cli": {
-      "version": {
-        "owner": "arduino",
-        "repo": "arduino-cli",
-        "commitish": "71a8576"
-      }
+      "version": "0.32.1"
```

```suggestion
      "version": "0.32.2"
```
A patch release was made to fix [a regression](https://github.com/arduino/arduino-cli/issues/2144) in the previous release (and the `71a8576` development version previously in use): 

https://github.com/arduino/arduino-cli/releases/tag/0.32.2